src/sparseml/exporters/transforms/utils/add_quantized_conv_matmul_add_ops.py

```diff
@@ -28,6 +28,44 @@
 __all__ = ["add_quantized_conv_matmul_add_ops"]
 
 
+def _get_axis(node, input_quantize_node):
+    """
+    return: the axis to be used for channel-wise QDQ -> Quantized
+        folding for Gemm, MatMul and Conv Ops
+    """
+    gemm_input_index_to_attribute = {
+        0: "transA",
+        1: "transB",
+    }
+
+    if node.op_type == "Conv":
+        return 0
+
+    if node.op_type == "MatMul":
+        # axis is always 1 for MatMul
+        return 1
+
+    axis = 1  # default axis is 1
+    target_id_index = int(input_quantize_node.name.split(".")[-2])
+    is_input = "input" in input_quantize_node.name
```

Comment:
this logic is specific to Q/DQs added by our sparsify one shot flow only - should instead find the index based on the DQ node - regardless, the weight will always be the "B" matrix so we shouldn't need this